src/DssSpell.sol

```diff
@@ -18,49 +18,261 @@ pragma solidity 0.6.12;
+            OPERATOR: 0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3,
+            INPUT_CONDUIT: 0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3,
+            OUTPUT_CONDUIT: 0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3,
+            URN:  0x0C185bf5388DdfDB288F4D875265d456D18FD9Cb,
+            gemID: "RWA006",
+            joinID: "MCD_JOIN_RWA006_A",
+            urnID: "RWA006_A_URN",
+            inputConduitID: "RWA006_A_INPUT_CONDUIT",
+            outputConduitID: "RWA006_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA006",
+            ilk: "RWA006-A",
+            ilk_string: "RWA006-A",
+            ilkRegistryName: "RWA006-A: Centrifuge: Alternative Equity Advisers",
+            RATE: TWO_PCT,
+            CEIL: 0 * MILLION,
+            PRICE: 0 * WAD,
+            MAT: 10_000,
+            TAU: 0,
+            DOC: ""
+        });
+
+        CentrifugeCollateralValues[4] memory collaterals = [RWA003, RWA004, RWA005, RWA006];
+
+        // integrate rwa003-006
+        for (uint i = 0; i < collaterals.length; i++) {
+            integrateCentrifugeCollateral(collaterals[i]);
+        }
+
+        // bump changelog version
+        DssExecLib.setChangelogVersion("1.9.2");
+    }
+
+    function integrateCentrifugeCollateral(CentrifugeCollateralValues memory collateral) internal {
+        address MIP21_LIQUIDATION_ORACLE =
+            DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+
+        address vat = DssExecLib.vat();
+
+        // Sanity checks
+        require(GemJoinAbstract(collateral.MCD_JOIN).vat() == vat, "join-vat-not-match");
+        require(GemJoinAbstract(collateral.MCD_JOIN).ilk() == collateral.ilk, "join-ilk-not-match");
+        require(GemJoinAbstract(collateral.MCD_JOIN).gem() == collateral.GEM, "join-gem-not-match");
+        require(GemJoinAbstract(collateral.MCD_JOIN).dec() == DSTokenAbstract(collateral.GEM).decimals(), "join-dec-not-match");
+
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
+            collateral.ilk, collateral.PRICE, collateral.DOC, collateral.TAU
+        );
+        (,address pip,,) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(collateral.ilk);
+
+        // Set price feed for RWA-00x
+        DssExecLib.setContract(DssExecLib.spotter(), collateral.ilk, "pip", pip);
+
+        // Init RWA-00x in Vat
+        Initializable(vat).init(collateral.ilk);
+        // Init RWA-00x in Jug
+        Initializable(DssExecLib.jug()).init(collateral.ilk);
+
+        // Allow RWA-00x Join to modify Vat registry
+        DssExecLib.authorize(vat, collateral.MCD_JOIN);
+
+        // Allow RwaLiquidationOracle to modify Vat registry
+        // DssExecLib.authorize(vat, MIP21_LIQUIDATION_ORACLE);
+
+        // Increase the global debt ceiling by the ilk ceiling
+        DssExecLib.increaseGlobalDebtCeiling(collateral.CEIL);
+        // Set the ilk debt ceiling
+        DssExecLib.setIlkDebtCeiling(collateral.ilk, collateral.CEIL);
```

Comment:
I'm guessing you forgot to pull in the `kovan` repo before you copied the spell over.  If you can think of other changes that would have been missed, please fix those too.  I think it's too late to just re-copy, as there have already been a bunch of changes to this PR.

```suggestion
        // set ilk/global DC
        DssExecLib.increaseIlkDebtCeiling(collateral.ilk, collateral.CEIL, true);
```

Reply:
I'm triple checking now, but looking at a diff between the two, it seems like the above was the only missed change.